datacatalog/cloud-client/src/test/java/com/example/datacatalog/LookupEntryBigQueryDatasetTest.java

```diff
@@ -0,0 +1,68 @@
+/*
```

Comment:
Tests don't need to be in separate files (unless you prefer them that way).

Reply:
Done.